admin/app/templates/admin_template.html

```diff
@@ -57,15 +57,15 @@
           {% endif %}
           {{ top_nav_item(url_for('main.support'), 'Support') }}
           {% if current_user.is_authenticated %}
-            {{ top_nav_item(url_for('main.documentation'), 'Documentation') }}
+            {{ top_nav_item(docs_base_url, 'Documentation') }}
```

Comment:
can these just all stay as what they were?

Reply:
What's your concern peter? I guess I see that it would link to the actual docs website URL rather than the internal one which always hits a redirect.

Reply:
concern was having 3 ways of doing the same thing. appreciate just having one obvious way to get there but having the direct link is fine too.

approved, go go go go